### PR TITLE
feat(link-list): aem improvements to end-of-section type

### PR DESCRIPTION
**Pull request by RaphaelAmadeu**
### Related Ticket(s)

#4902 

### Description

End variant of Link List was updated to reflect the new design specs by AEM team.

image

### CODE REVIEW

File: packages/web-components/src/components/link-list/link-list.scss [hunk 1]
```diff
@@ -95,4 +102,24 @@
...
+      grid-template-columns: 1fr 1fr 1fr;
+      grid-column-gap: 2rem;
+
+      ::slotted(#{$dds-prefix}-link-list-item) {
+        display: block;
+        margin-left: -$carbon--layout-01;
+        margin-right: 0;
+      }
+    }
+  }
```
Comment (asudoh) [inline packages/web-components/src/components/link-list/link-list.scss:124]: There seem many duplicate CSS rules with two columns mode. One way to solve it is:

```css
.#{$dds-prefix}-ce--link-list__list--split {
   /* The common rules */
}

.#{$dds-prefix}-ce--link-list__list--two-columns {
  /* 2-column mode-specific rules */
}

.#{$dds-prefix}-ce--link-list__list--three-columns {
  /* 3-column mode-specific rules */
}
```

```css
.#{$dds-prefix}-ce--link-list__list--split,
.#{$dds-prefix}-ce--link-list__list--three-columns {
   /* The common rules */
}
```

**Review by asudoh — COMMENTED** [review]
Great progress @RaphaelAmadeu!

**Comment by RobertaJHahn** [comment]
@RaphaelAmadeu @StephanieHoman is the designer on this work and should be assigned to review this PR.  Please update the design issue (#4900), letting her know that it is ready and provide the links where she can review it.

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @RaphaelAmadeu for the update!

**Review by jeffchew — CHANGES_REQUESTED** [review]
The documentation looks a little light as far as what the type means, and what options there are. Can you describe what types can be passed in, and what each type does?

Screen Shot 2021-01-24 at 10 02 34 PM

**Review by jeffchew — APPROVED** [review]

**Pull request merged**